README.md

```diff
@@ -83,23 +83,19 @@ A **Vagrant** configuration is included for development purposes. To use it, com
 - Install Vagrant and Virtualbox
 - Install the `vagrant-hostsupdater` plugin: `vagrant plugin install vagrant-hostsupdater`
 - Run `vagrant up`
-- Run `vagrant ssh -c "cd /vagrant && foreman start"`
+- Run `vagrant ssh -c "cd /vagrant && bin/dev"`
 - Open `http://mastodon.local` in your browser
 
 ### MacOS
 
 To set up **MacOS** for native development, complete the following steps:
 
-- Install the latest stable Ruby version (use a Ruby version manager for easy installation and management of Ruby versions)
-- Run `brew install postgresql@14`
-- Run `brew install redis`
-- Run `brew install imagemagick`
-- Run `brew install libidn`
-- Install Foreman or a similar tool (such as [overmind](https://github.com/DarthSim/overmind)) to handle multiple process launching.
-- Navigate to Mastodon's root directory and run `brew install nvm` then `nvm use` to use the version from .nvmrc
+- Use a Ruby version manager to install the specified version from `.ruby-version`
+- Run `brew install postgresql@14 redis imagemagick libidn` to install required libraries
```

Comment:
Nitpick, but `redis` and `postgresql` are not libraries. Also people might not want to install. postgresql@14` is they already have another postgres version installed (it will probably conflict).

Reply:
Updated to not call them libraries.

The `postgresql@14` I'm just pulling down from what was already there. With basically all of this, I sort of give some "if you know what you are doing, go ahead and do something else" leeway. Just want a nice happy path documented.